server/src/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java

```diff
@@ -666,18 +666,10 @@ public ConsoleProxyVO startNew(long dataCenterId) throws ConcurrentOperationExce
         return null;
     }
 
-    protected Map<String, Object> createProxyInstance(long dataCenterId, VMTemplateVO template) throws ConcurrentOperationException {
-
-        long id = _consoleProxyDao.getNextInSequence(Long.class, "id");
-        String name = VirtualMachineName.getConsoleProxyName(id, _instance);
-        DataCenterVO dc = _dcDao.findById(dataCenterId);
-        Account systemAcct = _accountMgr.getSystemAccount();
-
-        DataCenterDeployment plan = new DataCenterDeployment(dataCenterId);
-
+    protected NetworkVO getDefaultNetworkForCreation(DataCenter dc) {
```

Comment:
@ProjectMoon Could you please create a documentation and a test case for this method ("com.cloud.consoleproxy.ConsoleProxyManagerImpl.getDefaultNetworkForCreation(DataCenter)")?